src/LendingPool.sol

```diff
@@ -62,8 +62,24 @@ contract LendingPool is Guardian, TrustedCreditor, DebtToken, InterestRateModule
     mapping(address => address) public liquidationInitiator;
     mapping(address => mapping(address => mapping(address => uint256))) public creditAllowance; //mapping of vault => owner => beneficiary => amount
 
+    event TrancheAdded(address indexed tranche, uint8 indexed index, uint16 interestWeight, uint16 liquidationWeight);
+    event InterestWeightSet(uint256 indexed index, uint16 weight);
+    event LiquidationWeightSet(uint256 indexed index, uint16 weight);
+    event MaxInitiatorFeeSet(uint80 maxInitiatorFee);
+    event TranchePopped(address tranche);
+    event TreasuryInterestWeightSet(uint16 weight);
+    event TreasuryLiquidationWeightSet(uint16 weight);
+    event TreasurySet(address treasury);
+    event OriginationFeeSet(uint8 originationFee);
+    event BorrowCapSet(uint128 borrowCap);
+    event SupplyCapSet(uint128 supplyCap);
+    event Donate(address indexed from, address indexed tranche, uint256 amount);
     event CreditApproval(address indexed vault, address indexed owner, address indexed beneficiary, uint256 amount);
-    event Borrow(address indexed vault, bytes3 indexed referrer, uint256 amount);
+    event Borrow(
+        address indexed vault, address by, address indexed to, uint256 amount, uint256 fee, bytes3 indexed referrer
```

Comment:
Should we index 'by' instead of "to"? If we want to calculate the referral income/claims, it's searched/done for the user that took out the credit and not necessarily where it went to